### PR TITLE
[Test Automation] Verify RBAC plugin - REST API

### PR DESCRIPTION
**Pull request by Omar-AlJaljuli**
## Description

Test Automation that verifies all aspects of the RBAC REST API endpoints work.

## Which issue(s) does this PR fix

https://issues.redhat.com/browse/RHIDP-743

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Comment by josephca** [comment]
Hi @Omar-AlJaljuli, is this ready to be reviewed? I assume it's waiting to get e2e tests passed, right? FYI, I just updated the branch to rebase.

**Comment by Omar-AlJaljuli** [comment]
> Hi @Omar-AlJaljuli, is this ready to be reviewed? I assume it's waiting to get e2e tests passed, right? FYI, I just updated the branch to rebase.

Yes it is. It was just failing both e2e tests yesterday, but hopefully they pass after the rebase.

**Review by gustavolira — CHANGES_REQUESTED** [review]

File: e2e-tests/playwright/e2e/plugins/rbac/rbac.spec.ts [hunk 1]
```diff
@@ -10,6 +10,338 @@ import { Roles } from '../../../support/pages/rbac';
...
+      request =>
+        request.url() === '/api/search/query?term=' &&
+        request.method() === 'GET',
+    );
+    await uiHelper.openSidebar('Home');
+    const getRequest = await requestPromise;
+    authToken = await getRequest.headerValue('Authorization');
+  });
+
+  test('Test that roles and policies from GET request are what expected', async ({
```
[the remark this answers is not in the document]
Comment (Omar-AlJaljuli) [reply e2e-tests/playwright/e2e/plugins/rbac/rbac.spec.ts:39]: Do I put these functions in the Common.ts file or in the rbac.spec.ts file?

File: e2e-tests/playwright/e2e/plugins/rbac/rbac.spec.ts [hunk 2]
```diff
@@ -6,11 +6,338 @@ import {
...
+        },
+      );
+
+      expect(deleteRemainingPolicies.ok()).toBeTruthy();
+      expect(deleteRole.ok()).toBeTruthy();
+    },
+  );
+});
+
+test.describe.skip('Test RBAC plugin as an admin user', () => {
```
Comment (Omar-AlJaljuli) [inline e2e-tests/playwright/e2e/plugins/rbac/rbac.spec.ts:340]: I forgot to remove the .skip while testing, don't merge yet.

File: e2e-tests/playwright/e2e/plugins/rbac/rbac.spec.ts [hunk 3]
```diff
@@ -6,10 +6,337 @@ import {
...
+    );
+    await uiHelper.openSidebar('Home');
+    const getRequest = await requestPromise;
+    authToken = await getRequest.headerValue('Authorization');
+  });
+
+  test('Test that roles and policies from GET request are what expected', async ({
+    request,
+  }) => {
+    const expectedRoles =
```
Comment (gustavolira) [inline e2e-tests/playwright/e2e/plugins/rbac/rbac.spec.ts:43]: can you move expectedRoles and expectedPolicies to a functional class?

**Review by gustavolira — APPROVED** [review]

**Pull request merged**